content/en/apps/examples/direct-to-patient.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Direct-to-patient messaging workflows on CHT"
+linkTitle: "Direct-to-patient messaging workflows on CHT "
+weight:
+description: >
+  Reference for Direct-to-patient workflows with CHT and RapidPro
+relatedContent: >
+  apps/concepts
+  apps/reference/forms/app
+  apps/reference/tasks
+---
+
+{{% pageinfo %}}
+
+This documentation provides a guide for designing and deploying direct-to-patient messaging workflows to support patient care using CHT. It can serve as a great way to learn how to integrate CHT’s with technologies like [RapidPro](https://docs.communityhealthtoolkit.org/apps/features/integrations/rapidpro/) and SMS aggregators  to build direct-to-patient workflows.
```

Comment:
Some minor suggestions:
```suggestion
This documentation provides a guide for designing and deploying direct-to-patient messaging workflows to support patient care using the CHT. It can serve as an example for learning to integrate CHT apps with messaging apps, like [RapidPro](https://docs.communityhealthtoolkit.org/apps/features/integrations/rapidpro/), and SMS aggregators to build direct-to-patient workflows.
```